# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/tenacity/hotfix-gcc15.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- tenacity/libraries/lib-utility/MemoryX.cpp.vanilla	2025-04-29 00:45:01.199109946 +0300
+++ tenacity/libraries/lib-utility/MemoryX.cpp	2025-04-29 00:45:20.535519143 +0300
@@ -10,6 +10,8 @@
 
 #include "MemoryX.h"
 
+#include "stdint.h"
+
 // Make the symbol table non-empty
 UTILITY_API void lib_utility_dummy_symbol()
 {}
